app/models/spree/user.rb

```diff
@@ -47,6 +48,10 @@ def self.admin_created?
       User.admin.count > 0
     end
 
+    def self.from_omniauth(auth)
+      ##
+    end
```

Comment:
We'll need to fill out this method of course, and have a migration to add a field (or fields) to the User model for the OIDC identifier.